tasks/fetch_ward_data.py

```diff
@@ -52,10 +53,28 @@ def run(self):
         repository.store_dataframe(all_wards, ward_storage_location, allow_create=True, store_index=True)
 
         # impute delta's atleast for Mumbai this is needed it only provides totals
+        delta_needed_for = ['total.confirmed', 'total.recovered', 'total.deceased', 'total.active']
+        group_by_cols = ['state', 'district', 'ward']
+        all_wards = create_delta_cols(all_wards,group_by_cols, delta_needed_for)
+
         all_wards.to_csv(self.output().path, index=True)
 
+
     def output(self):
         return luigi.LocalTarget(f'wards_{date.today()}.csv')
 
     def complete(self):
         return self.output().exists()
+
+
+def create_delta_cols( df,group_by_cols, delta_needed_for_cols):
```

Comment:
move this to the backend package we're trying to keep the tasks as light as possible.
All the smart stuff can go to /backend/
this one could move to maybe /backend/data/utility.py perhaps...